e2e/opi-nvme.go

```diff
@@ -0,0 +1,111 @@
+package e2e
+
+import (
+	"time"
+
+	"github.com/ShellCode33/VM-Detection/vmdetect"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	opiNvmeConfigMapData = `{
+  "nodes": [
+    {
+      "name": "localhost",
+      "rpcURL": "http://127.0.0.1:9009",
+      "targetType": "nvme-tcp",
+      "targetAddr": "127.0.0.1"
+    }
+  ]
+}`
+)
+
+var _ = ginkgo.Describe("SPDKCSI-OPI-NVME", func() {
+	f := framework.NewDefaultFramework("spdkcsi")
+	ginkgo.BeforeEach(func() {
+		deployConfigs(opiNvmeConfigMapData)
+		deployOpiNvmeConfig()
+		deployCsi()
+	})
+
+	ginkgo.AfterEach(func() {
+		deleteCsi()
+		deleteOpiNvmeConfig()
+		deleteConfigs()
+	})
+
+	ginkgo.Context("Test SPDK CSI OPI NVME", func() {
```

Comment:
@yanjing1104 Let's not reduplicate the e2e code for OPI, instead i suggest the same existing tests should work for different XPU backends. We should only run the existing tests for both the backends.